docs/index.md

```diff
@@ -43,6 +43,13 @@ We expect volunteers to avoid sharing their personal political preferences. We e
 
 #### Chicago Mayoral Race
 
+#### When
+* [Tuesday, February 26, 2019 is the date of the Chicago mayoral election. A run-off election may be held Tuesday, April 2, 2019 in the event that there is no clear majority of votes for the candidates. Source: Chicago Board of Election Commissioners]
```

Comment:
There is a space between the ']' and the '('

This causes the square brackets to render instead of it being a link.